### PR TITLE
add grade parsing

### PR DESCRIPTION
**Pull request by ros3bud**
#12 

Test, method & response vars added.

### CODE REVIEW

**Comment by ros3bud** [comment]
@cgrayson, LGTY?

File: src/outbound.coffee [hunk 1]
```diff
@@ -101,6 +102,7 @@ response.variables = ->
     { name: 'outcome', type: 'string', description: 'The outcome of the lookup will be success if a credit score is returned, and failure if not. An outcome of Error indicates a problem was encountered during lookup.' }
     { name: 'reason', type: 'string', description: 'The description for a failure or error outcome.' }
     { name: 'zip4_credit_score', type: 'number', description: 'Experian generated median credit score.' }
+    { name: 'zip4_credit_score_grade', type: 'string', description: 'Score based credit grade. Possible outcomes are: Super Prime, Prime, Near Prime, Non-Prime, and High Risk' }
```
Comment (alexkwolfe) [inline src/outbound.coffee]: I would recommend showing the actual score ranges next to the grade name:

`Score based credit grade: Super Prime (740+), Prime (680–739), Near Prime (620–679), Non-Prime (550–619), or High Risk (<550)`

**Comment by ros3bud** [comment]
Ok, that makes sense.

**Comment by ros3bud** [comment]
@alexkwolfe, updated.

**Comment by alexkwolfe** [comment]
I'll let @cgrayson look at it. ;)

**Comment by cgrayson** [comment]
LGTM

**Comment by cgrayson** [comment]
_(testing Slack integration)_

**Pull request merged**